Start of work on draft -10.

jhaas-pfrc opened the pull request:
Put back AS_CONFED_SET.
Lots of word smithing.
Lots of xml and indentation cleanup.
Remove comments for things that have been addressed or otherwise accepted. Add two appendices: One covers the filtering example and moves the example out of the normative text. The second discusses inconsistent origin AS issues.

jhaas-juniper committed:
Start of work on draft -10.

Put back AS_CONFED_SET.
Lots of word smithing.
Lots of xml and indentation cleanup.
Remove comments for things that have been addressed or otherwise accepted.
Add two appendices: One covers the filtering example and moves the example out
of the normative text. The second discusses inconsistent origin AS issues.

wkumari commented on draft-ietf-idr-deprecate-as-set-confed-set.xml:
```diff
@@ -105,27 +105,34 @@ title) for use on http://www.rfc-editor.org/rfcsearch.html.
+      in the AS_PATH.  This is done to simplify the design and implementation
+      of BGP and to make the semantics of the originator of a BGP route clearer.
+      This will also simplify the design, implementation, and deployment of
+      various BGP security mechanisms. This document, if approved, updates RFC
```
I'd suggest instead: 
"This document updates RFC 4271 and RFC 5065, and obsoletes RFC 6472." and then add "(if approved)" to the Updates: header.

The reason for this is that it would be embarrassing if we forgot this text in the final RFC version.

jhaas-pfrc replied:
I've made the text change. (See latest diff.)
I'm not quite sure what you're asking about for the Updates: header.  The (if approved) text was present in that last diff.  See *10.txt which contains the compiled document.

I'll be waiting on approval from Sriram before doing merge and then let him do the publish to IETF.

wkumari replied:
Whoops, apologies, I hadn't checked the rendered output and had forgotten that XML2RFC / etc automagically add "(if approved)" (e.g: "Updates: 4271 5065 (if approved)").

So, we are just waiting on Sriram to submit I think...

wkumari reviewed (approved):
I've made one (non-blocking / editorial) suggestion.

Because it's a minor note, I decided it was cleaner to just put it as a comment - feel free to address or not.

Thank you very much, kind sir...

jhaas-juniper committed:
Minor tweak from Warren to the 'this document updates' text

ksriram25 committed:
Update draft-ietf-idr-deprecate-as-set-confed-set.xml

ksriram25 committed:
Made the corrections pointed out by Jeff

I made this change in the Security Section:
s/This document obsoletes the use of aggregation techniques/This document deprecates the use of aggregation techniques/. 

I have added this new paragraph to the Recommendations (Section 3):
 
      The document uses normative language such as "SHOULD NOT send" 
      rather than "MUST NOT send" with the intention of allowing 
      some transition time for exisiting implementations and
      avoiding abrupt disruptions for the operators currently
      using AS_SETs or AS_CONFED_SETs. 
      However, it is strongly urged that operators stop
      sending UPDATEs with AS_SETs or AS_CONFED_SETs as quickly as 
      possible to avoid having UPDATEs dropped by BGP security 
      mechanisms such as RPKI-ROV and BGPsec.

ksriram25 committed:
Update draft-ietf-idr-deprecate-as-set-confed-set.xml

jhaas-pfrc committed:
Merge pull request #5 from jhaas-pfrc/ksriram25-patch-3

Made the corrections pointed out by Jeff

jhaas-pfrc committed:
Merge pull request #3 from jhaas-pfrc/ksriram25-patch-2

Update draft -- Sriram's edits

jhaas-pfrc committed:
Change one instance of aggregator to aggregating

jhaas-pfrc commented:
Confirmed Sriram's desire to merge in email.
